src/Runner.Common/ConfigurationStore.cs

```diff
@@ -119,8 +119,11 @@ public interface IConfigurationStore : IRunnerService
         CredentialData GetCredentials();
```

Comment:
this is basic add things back from https://github.com/actions/runner/pull/513/files#diff-8ef9a7b7967d070375a6e4773f2bbeb1ae60e986bff48a4171256ae1b3869872

and introduce migrated config file (.runner) in addition to the migrated credentials file (.credentials).